web/lib/HabitatFlipCard.js

```diff
@@ -91,8 +89,17 @@ class HabitatFlipCard extends HTMLElement {
     for (const e of this.shadowRoot.querySelectorAll('.icon-info')) {
       e.addEventListener('click', () => {
         const wrapper = this.shadowRoot.querySelector('.flip-wrapper');
+        //set front w/h dimensions to back
+        this.parentElement.style.setProperty(`--backWidth`, `${this.shadowRoot.querySelector('.flip-card-front').clientWidth}px`, `important`);
+        this.parentElement.style.setProperty(`--backHeight`, `${this.shadowRoot.querySelector('.flip-card-front').clientHeight}px`, `important`);
+        //and use --backheight / --backwidth css vars
+
         const flipped = wrapper.classList.toggle('flip');
         if (flipped) {
+          window.addEventListener('resize', () => {
```

Comment:
move this inside the constructor. Otherwise we add a new event listener on each click if the card is flipped